frontend/src/apis/books/BooksAPI.tsx

```diff
@@ -220,12 +223,14 @@ export const BOOKS_API = {
     if (req.book.pageCount) {
       formData.append("pageCount", req.book.pageCount.toString());
```

Comment:
Just need to check with @hkder here, should we be sending 0 in the form data to set to null, or should we be omitting from the form data to set to null? We are currently doing one in book add, one in book modify. Not sure if you've changed this, or if they're still different.